cluster/addons/addon-manager/Makefile

```diff
@@ -30,6 +30,10 @@ endif
 ifeq ($(ARCH),ppc64le)
 	BASEIMAGE?=ppc64le/debian
 endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE?=s390x/python:2.7-slim
```

Comment:
Why not `s390x/debian` like the other arches?
If you want to refactor this then to using other base images, it's fine, but not in this PR